gms/gossiper.cc

```diff
@@ -582,9 +582,9 @@ future<> gossiper::do_apply_state_locally(gms::inet_address node, const endpoint
                     // apply states, but do not notify since there is no major change
                     co_await this->apply_new_states(node, local_state, remote_state, permit.id());
                 } else {
-                    logger.trace("Ignoring remote version {} <= {} for {}", remote_max_version, local_max_version, node);
+                    logger.debug("Ignoring remote version {} <= {} for {}", remote_max_version, local_max_version, node);
                 }
-                if (!local_state.is_alive() && !this->is_dead_state(local_state)) { // unless of course, it was dead
```

Comment:
We don't need to check `this->is_alive(node)` in place of the previous check?

Reply:
It shouldn't be a problem to call mark_alive on an endpoint that's already alive
but saving that makes sense.